README.md

```diff
@@ -51,10 +74,13 @@ Command      | Description
 backtrace    | Show backtrace.
 bindings     | Show binded variables.
 continue     | Continue the execution of the process.
-finish       | Finis the debug session.
+finish       | Finish the debug session.
 messages     | Show process messages.
 next         | Execute next instruction.
 skip         | Skip.
 step         | Step inside next instruction.
 stop         | Stop debugging.
 timeout      | Timeout.
+Var          | Variable names are evaluated in the current context.
+
+Abbreviations of the above commands are also accepted, like `n` for `next`, `ba` for `backtrace` etc.
```

Comment:
could you add some docs to show how to use the MFA version to add breakpoints? Thanks